docs/pages/iconography_and_svgs.js

```diff
@@ -5,28 +5,29 @@ import MainSection from '../components/MainSection.js';
 import Page from '../components/Page.js';
 import PageHeader from '../components/PageHeader.js';
 
+const { icons } = Icon;
 type IconName = $NonMaybeType<$ElementType<ElementProps<typeof Icon>, 'icon'>>;
 
 function ClickableIcon({ iconName, onTap }: {| iconName: IconName, onTap: () => void |}) {
   return (
-    <Tooltip text={String(iconName)} accessibilityLabel="">
+    <Tooltip text={iconName} accessibilityLabel="">
       <TapArea rounding="circle" tapStyle="compress" onTap={onTap}>
         <Box padding={2}>
-          <Icon color="darkGray" accessibilityLabel={String(iconName)} icon={iconName} />
+          <Icon color="darkGray" accessibilityLabel={iconName} icon={iconName} />
         </Box>
       </TapArea>
     </Tooltip>
   );
 }
 
+function findIcon(icon?: string): ?IconName {
```

Comment:
It seems like this is the crux of the fix to remove the stringification and .map call?